File: python/AschiChar.py
def Aschichar(arr):
    for ch in arr:
        if "0"<=ch<="9":
            continue
        if "a"<=ch<="z":
            continue
        if "A"<=ch<="Z":
            continue
        return False
    return True

print(Aschichar("jhdsjkaio3534@"))

File: JavaScript/AschiChar.js
function AschiChar(arr){
    for(let i=0; i<arr.length; i++){
        let ch= arr[i]
        if(ch>="0" && ch<="9") continue
        if(ch>="a" && ch<="z") continue
        if(ch>="A" && ch<="Z") continue
        return false
    }
    return true
}
console.log(AschiChar("kjsdfhjkdshfiorwqi432985239@"))